Gogacz-ReportService

StrongMen120 opened the pull request:
Gogacz-ReportService dodanie metod

StrongMen120 committed:
Gogacz-ReportService

Gogacz-ReportService

Jerry-BloodBerry reviewed (approved):
Sprawdziłem, wszystko w porządku. Można mergować 👍 (Nieaktualne)

Jerry-BloodBerry reviewed (changes requested):
Dlaczego room jest zmieniony z string na int a _reportPositions usunięte?

MikolajczukKamil commented:
`ref` jakie dodawał VP trzeba jeszcze wywalić, gdzieniegdzie się zapodziały te refy

Jerry-BloodBerry commented:
Trzeba też dodać konstruktor przyjmujący parametr typu APIController i ustawiający pole ApiController.

StrongMen120 committed:
Gogacz-ReportService

Dodany Konstruktor usunięte ref
Room zmieniłem na int ponieważ z bazy dostaje wartość int
_reportPositoion usunięte bo w bazie nie mam takich danych

Jerry-BloodBerry committed:
Usunięcie ostatniego ref

Jerry-BloodBerry reviewed (approved):
Wszystko okej, można mergować 👍
